app/uk/gov/hmrc/gform/views/govuk_wrapper.scala.html

```diff
@@ -37,7 +37,8 @@
   scriptElem: Option[Html] = None,
   authConfig: Option[AuthConfig],
   navLinks: Option[Html] = None,
-  includeHMRCBranding: Boolean
+  includeHMRCBranding: Boolean,
+  templateId: String
```

Comment:
Let's keep type information here ie. use `FormTemplateId` instead of `String`

Reply:
Done